symfony/plugins/orangehrmAdminPlugin/Dao/PayGradeDao.php

```diff
@@ -151,4 +184,137 @@ public function getCurrencyById(string $id): ?CurrencyType
             throw new DaoException($e->getMessage(), $e->getCode(), $e);
         }
     }
+
+    /**
+     * @param PayGradeSearchFilterParams $payGradeSearchFilterParams
+     * @return int
+     * @throws DaoException
+     */
+    public function getPayGradesCount(PayGradeSearchFilterParams $payGradeSearchFilterParams):int
+    {
+        try{
+            return $this->getPayGradesPaginator($payGradeSearchFilterParams)->count();
+        }catch (Exception $e){
+            throw new DaoException($e->getMessage());
+        }
+    }
+
+    /**
+     * @param PayGrade $payGrade
+     * @return PayGrade
+     * @throws DaoException
+     */
+    public function savePayGrade(PayGrade $payGrade): PayGrade
+    {
+        try {
+            $this->persist($payGrade);
+            return $payGrade;
+        } catch (Exception $e) {
+            throw new DaoException($e->getMessage());
+        }
+    }
+
+    /**
+     * @param PayGradeCurrency $payGradeCurrency
+     * @return PayGradeCurrency
+     * @throws DaoException
+     */
+    public function savePayGradeCurrency(PayGradeCurrency $payGradeCurrency): PayGradeCurrency
+    {
+        try {
+            $this->persist($payGradeCurrency);
+            return $payGradeCurrency;
+        } catch (Exception $e) {
+            throw new DaoException($e->getMessage());
+        }
+    }
+
+    /**
+     * @param array $tobeDeletedIds
+     * @return int
+     * @throws DaoException
+     */
+    public function deletePayGrade(array $tobeDeletedIds): int
```

Comment:
seems this function delete many paygrades

Reply:
rename function to plural